swagger/api/swagger/swagger.yaml

```diff
@@ -76,20 +78,20 @@ paths:
           schema:
             $ref: '#/definitions/Error'
             
-  /{ownerId}/layer/{layerId}/{instanceId}/:
+  /{ownerId}/layer/{layerId}/instance/{instanceId}/:
```

Comment:
I'm not sure if all these paths should start with owner or not:

https://wxtiles.com/metocean/layer/awsome-layer/instance/77th-Run/
vs.
https://wxtiles.com/owner/metocean/layer/awsome-layer/instance/77th-Run/